ui/components/ui/show-hide-toggle/index.scss

```diff
@@ -6,6 +6,7 @@
     appearance: none;
 
     + .show-hide-toggle__label {
+      display: flex;
```

Comment:
Adding display flex here updates the component icon aligment

### Before

![Screenshot 2024-03-05 at 7 25 57 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/73785eb2-065f-4c5d-8b99-4e62207f27e5)

### After

![Screenshot 2024-03-05 at 7 27 18 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/92300fa4-4a6f-4228-a545-4fd26c0c7c22)

Reply:
thank you for the change. i will resolve this conversation.